ui/components/multichain/notification-detail-copy-button/notification-detail-copy-button.tsx

```diff
@@ -69,9 +70,10 @@ export const NotificationDetailCopyButton: FC<
         properties: {
           notification_id: notification.id,
           notification_type: notification.type,
-          ...('chain_id' in notification && {
```

Comment:
I don't understand this change, seems simpler to check for `chain_id`?

Reply:
Meant to revert this, good catch.